reddit.js

```diff
@@ -1,5 +1,30 @@
 var reddit = require("./data.json")
 
 // Write your solutions below
+// var titles = reddit.data.children.data.title;
+var titles = reddit.data.children;
```

Comment:
Here, perhaps 'posts' or 'children' would be a more appropriate variable name than titles, just so that you or other programmers don't confuse it with the actual title of the post.